atd-vze/src/Components/DataTable.js

```diff
@@ -166,7 +166,7 @@ const DataTable = ({
                                   <Input
                                     name={field}
                                     id={field}
-                                    onChange={e => handleInputChange(e)}
+                                    onClick={e => handleInputChange(e)}
```

Comment:
I noticed that when you go to a temp record without an onsys value yet and click edit and select "Yes" and save, it wasnt saving. You had to first click No and then Yes for it to log the value and save. Changing this to `onClick` allows your first click to be logged as the new value. Its still somewhat of a confusing behavior that you have to click at all when Yes already looks to be selected in the UI so im trying to think through other solutions..

Reply:
Good catch! I took a small dive into this, and I think you have uncovered another example where users cannot clear out a value. They can only select `Y` or `N` in this case and are unable to clear that out once chosen. It seems like the input is taking the `null` value and and just going with `Y` which is why the `onChange` handler isn't firing (since it appears as no change has happened when the value goes `Y` => `Y`). 

Could be a spin-off issue, but I think that adding another option to the select like `<option value={null}></option>` or something would patch this up.

Reply:
yeahh i agree that it would be good to add a null/clear option. @patrickm02L another issue or address that here?

Reply:
@roseeichelmann Could be a good issue to write up and bring to backlog refinement tomorrow!